docs/source/getting-started.rst

```diff
@@ -1282,6 +1284,9 @@ Maven repositories:
 \*\* ``[owner]`` should be the owner of the  repository and ``[repo]`` should
      be the name of the Bintray repository.
 
+By default, ``epm`` tries to resolve your dependencies against the
+above repositories. So, you don't need to specify a repository unless required.
```

Comment:
Should be '..resolve your dependencies against ``central``'. The other repositories specified in the table don't get used unless you specify them.